### ISSUE TITLE
Disable spell check

### ISSUE THREAD

**Issue by unm07** [IEEE-VIT/RazorType#22]
Disable spell check when user inputs wrong spelling

**Comment by unm07** [comment]
May I work on this?

**Comment by KaviyaKumarr** [comment]
Yes I've assigned the issue to you so you can work on it but kindly ensure to give updates.

**Comment by sakshipatil4403** [comment]
@KaviyaKumarr pls assign me this issue

**Comment by unm07** [comment]
@sakshipatil4403  I have unassigned myself 
you can ask the moderators to assign you
